src/azure-cli/azure/cli/command_modules/botservice/tests/latest/test_bot_commands.py

```diff
@@ -770,6 +769,42 @@ def __talk_to_bot(self, message_text='Hi', expected_text=None):
                                 expected_text)
 
 
+class BotLiveOnlyTests(LiveScenarioTest):
```

Comment:
May I know why you have such live only test?

Reply:
This is to address issue discussed at https://github.com/Azure/azure-cli/pull/8384#discussion_r368392620

Reply:
This is to address the issue discussed at https://github.com/Azure/azure-cli/pull/8384#discussion_r368392620

Reply:
@Juliehzl, @jiasli is correct, it's to address the issue raised in #8384.